core/services/workflows/engine_test.go

```diff
@@ -78,6 +80,25 @@ type testHooks struct {
 	executionFinished chan string
 }
 
+func newTestDBStore(t *testing.T, clock clockwork.Clock) store.Store {
```

Comment:
I added this so we can see a workflow id field in test logs, it was just empty before which made it awkward to read test logs. When the wfid was empty, it would pass the fk constraint, so i had to add this function to make sure that the constraint stays valid with a non empty workflow id